modules/flowable-rest/src/test/java/org/flowable/rest/service/api/runtime/SignalsResourceTest.java

```diff
@@ -141,8 +141,7 @@ public void testSignalEventReceivedAsync() throws Exception {
             var.put("value", "test");
 
             httpPost.setEntity(new StringEntity(requestNode.toString()));
-            closeResponse(executeRequest(httpPost, HttpStatus.SC_BAD_REQUEST));
-
+            closeResponse(executeRequest(httpPost, HttpStatus.SC_ACCEPTED));
```

Comment:
Add assertions that the variables were correctly saved and that continuing will use them properly